src/Service.Host/client/src/components/reports/CartonDetails.js

```diff
@@ -12,38 +12,32 @@ const styles = () => ({
     }
 });
 
-class CartonDetails extends Component {
-    render() {
-        const { reportData, loading, classes } = this.props;
-
-        return (
-            <Paper className={classes.root}>
-                <Grid container spacing={24} justify="center">
-                    <Grid item xs={12}>
-                        <Typography variant="h4" gutterBottom>
-                            Carton Details
-                        </Typography>
-                    </Grid>
-                    <Grid item xs={12}>
-                        <Link style={{ display: 'block' }} to="/products/maint/carton-types/create">
-                            <Typography>Create new carton type</Typography>
-                        </Link>
-                    </Grid>
-                    <Grid item xs={12}>
-                        {loading ? <Loading /> : ''}
-                        <ReportTable
-                            reportData={reportData}
-                            showTotals={false}
-                            placeholderRows={10}
-                            placeholderColumns={3}
-                            showTitle={false}
-                        />
-                    </Grid>
-                </Grid>
-            </Paper>
-        );
-    }
-}
+const CartonDetails = ({ reportData, loading, classes }) => (
+    <Paper className={classes.root}>
+        <Grid container spacing={24} justify="center">
+            <Grid item xs={12}>
+                <Typography variant="h4" gutterBottom>
+                    Carton Details
+                </Typography>
+            </Grid>
+            <Grid item xs={12}>
+                <Link style={{ display: 'block' }} to="/products/maint/carton-types/create">
+                    <Typography>Create new carton type</Typography>
+                </Link>
+            </Grid>
+            <Grid item xs={12}>
+                {loading ? <Loading /> : ''}
```

Comment:
Would this be better as 
`{loading && <Loading />}`

Reply:
yes that would be better. I'll change it.